tests/snuba/api/endpoints/test_organization_events_stats.py

```diff
@@ -252,62 +252,26 @@ def test_throughput_epm_hour_rollup(self):
                     project_id=project.id,
                 )
 
-        with self.feature("organizations:discover-basic"):
-            response = self.client.get(
-                self.url,
-                format="json",
-                data={
-                    "start": iso_format(self.day_ago),
-                    "end": iso_format(self.day_ago + timedelta(hours=6)),
-                    "interval": "1h",
-                    "yAxis": "epm()",
-                    "project": project.id,
-                },
-            )
-        assert response.status_code == 200, response.content
-        data = response.data["data"]
-        assert len(data) == 6
-
-        rows = data[0:6]
-        for test in zip(event_counts, rows):
-            assert test[1][1][0]["count"] == test[0] / (3600.0 / 60.0)
-
-    def test_throughput_epm_day_rollup(self):
```

Comment:
Why is the day rollup test deleted here?

Reply:
🤨 oh, didn't even notice I did that, thanks for the catch.